drivers/modem/Kconfig.ehs6

```diff
@@ -0,0 +1,65 @@
+# Kconfig - Gemalto EHS6 driver options
+
+#
+# Copyright (c) 2018 Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#CONFIG_OFFLOAD_SOCKET
+
+menuconfig MODEM_EHS6
+	bool "Enable Gemalto EHS6 modem driver"
+	depends on MODEM
+	select MODEM_RECEIVER
+	select NET_OFFLOAD
```

Comment:
I believe you'll also want to also select  `NET_SOCKETS_OFFLOAD` as you're using those APIs as well as the `NET_OFFLOAD` APIs.

Reply:
That's right.